src/api/socket.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  CommandMethods,
+  CommandParams,
+  CommandResult,
+  EventMethods,
+  EventParams,
+  PauseId,
+  ProtocolClient,
+  SessionId,
+} from "@replayio/protocol";
+import WebSocket from "ws";
+
+import { defer } from "./utils";
+
+const address = "wss://dispatch.replay.io";
```

Comment:
This should be parameterized so that it will work in cloud-dev and ephemeral environments. We have existing environment variables for this